std_msgs/Header header
ttl_driver/DxlMotorHardwareStatus[] motors_hw_status
